[HOTFIX]

joannestephany opened the pull request:
## Feito
- Alteração de appBar para proporção real em todas as telas
- Reposicionamento de itens
- Alteração de cor ( o cinza que deixei azul pode ser mudado, sem problema)
- Adicionando cor como característica das disciplinas ( cores podem ser geradas aleatoriamente, se for fazer associado à disciplina será necessário adicionar atributo ao banco de dados)
- Adição de icones: download, check ( para quando atividade estiver com status enviada), calendario (para label de data), relogio (para label de horario)


## Pendencias
- Rever padrão de listagem, decidir entre caixa ou borda infinita.
- Revisar a tela_020
- Revisar se todas as fontes estão com raleway
- revisar alinhamento de tudo

joannestephany committed:
[HOTFIX]

## Feito
- Alteração de appBar para proporção real em todas as telas
- Reposicionamento de itens
- Alteração de cor ( o cinza que deixei azul pode ser mudado, sem problema)
- Adicionando cor como característica das disciplinas ( cores podem ser geradas aleatoriamente, se for fazer associado à disciplina será necessário adicionar atributo ao banco de dados)
- Adição de icones: download, check ( para quando atividade estiver com status enviada), calendario (para label de data), relogio (para label de horario)


##Pendencias: 
- Rever padrão de listagem, decidir entre caixa ou borda infinita.
- Revisar a tela_020
- Revisar se todas as fontes estão com raleway
- revisar alinhamento de tudo

KevennyJS reviewed (approved):
Alterações aceitas.

KevennyJS reviewed (approved):
Revisão realizada.

WalterHenri reviewed (commented):
eu ate ia reclamar mas vc deve saber o que faz. 
eu tava acostumado com o header grande :(

WalterHenri reviewed (approved):
bom.

WalterHenri reviewed (approved):
bom.

joannestephany commented:
Já que ainda não foi mergeado, estou fazendo commit por cima

joannestephany committed:
[HOTFIX]

## FEITO
- Revisada a tela_020
- Ajuste no alinhamento dos botões voltar e confirmar, padrão ficar em rodapé da página
- appBar diferenciado entre professor e alunos ( só alterei o nome para deixar claro, pois quando houver a sidebar elas tem navegações diferentes)
